src/gtimelog/settings.py

```diff
@@ -40,6 +40,7 @@ class Settings(object):
     virtual_midnight = datetime.time(2, 0)
 
     task_list_url = ''
+    auth_header = ''
```

Comment:
This doesn't belong here.

It's my bad; this module (and the class) ought to be renamed legacy_settings.py (and LegacySettings).  It's only here to support old setups that had a ~/.gtimelog/gtimelogrc file before the settings got moved into gsettings, which happened a long time ago.